apps/mobile-alphabet/src/store/slices/alphabet-slice.ts

```diff
@@ -0,0 +1,21 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+
+type AlphabetApiDataPayload = {
+    endpoint: string;
+    data: JSON;
+};
+
+const alphabetApiSlice = createSlice({
+    name: 'alphabetApi',
```

Comment:
You might want to export a constant for this, although it's probably overkill for now.